src/base-input.js

```diff
@@ -155,12 +155,17 @@ export default class BaseInput extends LitElement {
    * @private Validates value of the input in blur
    * @returns {string} Validates string entered into the input field.
    */
-  handleInput(el) {
-    this.inputValue = el.target.value;
-
+  handleInput(e) {
```

Comment:
Is `e` still needed? This value of this is not being used or being passed in? 

Locally, I removed it and it all seemed to work?

Reply:
Yep, it can be removed. Fixed.